docs/src/add-services/kafka.md

```diff
@@ -51,7 +51,8 @@ title=Ruby
 file=none
 highlight=ruby
 markdownify=false
----
++++
+
```

Comment:
:clipboard: This adds an extra blank line since this isn't being parsed as Markdown.
```suggestion
```